proxy/config.go

```diff
@@ -128,3 +129,21 @@ func AddService(filepath string, service Service) error {
 	_, err := file.WriteString(serviceStr)
 	return err
 }
+
+// RemoveService remves a service from the filepath
+func RemoveService(filepath string, service Service) error {
+	file, err := ioutil.ReadFile(filepath)
+
+	if err != nil {
```

Comment:
Could we have it tested as well? 

Pass an invalid file path to it like `RemoveService('foobarinvalid', ...)`

Reply:
Yes, yes we can. I haven't thought about testing invalid files as input. Great catch.